docs/src/virtual_machines/using_shared_disks.md

```diff
@@ -0,0 +1,48 @@
+# Shared disks
```

Comment:
Add document to the summary, that way it will be accessible. Currently user can not access the document if the path is not known.